src/attachment/style.ts

```diff
@@ -186,4 +186,18 @@ export const ATTACHMENT_CSS = `
     color: #3498db;
     font-size: 14px;
   }
+  .kuc-attachment__group__files__browse-button {
```

Comment:
you don't need to write css for class `.kuc-attachment__group__files__browse-button` here, because you already wrote it above before